src/main/java/seedu/address/model/client/ClientId.java

```diff
@@ -20,6 +21,37 @@ public ClientId(int id) {
         this.clientId = id;
     }
 
+    public int getIdInt() {
+        return this.clientId;
+    }
+
+    /**
+     * Checks if this ClientID is empty.
+     * @return false since the ClientID is not empty.
+     */
+    public boolean isEmpty() {
+        return false;
+    }
+
+    /**
+     * Represents an Empty ClientID.
+     */
+    public static class EmptyClientId extends ClientId {
+        public static final ClientId EMPTY_CLIENT_ID = new EmptyClientId();
+        public EmptyClientId() {
+            super(3);
```

Comment:
Perhaps, -1 or some other negative number? (again, 3 could be a valid index at some point)

Reply:
Resolved